experimental/babel-preset-env/src/index.js

```diff
@@ -111,12 +111,25 @@ const getPlatformSpecificDefaultFor = (targets: Targets): ?Array<string> => {
   return isAnyTarget || isWebTarget ? defaultWebIncludes : null;
 };
 
+const getOptionSpecificExcludesFor = ({
```

Comment:
Should we move this into `./default-includes.js` and maybe rename it `defaults.js`?